tests/test_pylast.py

```diff
@@ -23,6 +23,7 @@ def load_secrets():
         try:
             doc["username"] = os.environ['PYLAST_USERNAME'].strip()
             doc["password_hash"] = os.environ['PYLAST_PASSWORD_HASH'].strip()
+            doc["token"] = os.environ['PYLAST_TOKEN'].strip()
```

Comment:
I don't think any value will work reliably because the tokens are only valid once. I'm going to rewrite the test to simply check that the token is validated and Last.fm returns an invalid token message.